src/main/views/case-standards-list.njk

```diff
@@ -0,0 +1,104 @@
+{% from "./macros/common-components.njk" import goBack %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ goBack(text = backButton) }}
+{% endblock %}
+{% block content %}
+  <div class="parent-box">
+    <h2 id="page-heading" class="govuk-heading-l">
+      {{ heading }}<br>{{ heading2 }}
+    </h2>
+    <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-bottom-1">{{ listDate }} {{ contentDate }}</p>
+    <p class="govuk-body">{{ listUpdated | replace("DATE", publishedDate) }} {{ publishedTime }}</p>
+
+    <details class="govuk-details" data-module="govuk-details" open>
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">{{ importantInformationHeading }}</span>
+      </summary>
+      <div class="govuk-details__text">
+        <p>{{ importantInformationP1 }}</p>
+        <p>{{ importantInformationP2 | replace("EMAIL", venueEmail) }}</p>
+        <p>{{ importantInformationP3 }}</p>
+        {% if bill %}
+          <P>{{ importantInformationP4 }}</P>
+          <P>{{ importantInformationP5 | replace("EMAIL", venueEmail) }}</P>
+          <P>{{ importantInformationP6 }}</P>
+        {% endif %}
+      </div>
+    </details>
+
+    <div class="parent-box overflow-table">
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">{{ hearingDate }}</th>
+          <th scope="col" class="govuk-table__header">{{ caseName }}</th>
+          <th scope="col" class="govuk-table__header">{{ duration }} </th>
+          <th scope="col" class="govuk-table__header">{{ hearingType }}</th>
+          <th scope="col" class="govuk-table__header">{{ venue }}</th>
+        </tr>
+        </thead>
+        <tbody class="govuk-table__body">
+        {% for courtCase in listData %}
+          {% set duration = '' %}
+          {% if courtCase['durationAsDays'] !== 0 %}
```

Comment:
Wondering if this duration calculation should be moved into the JS (similar to 1373)

Reply:
Done. While doing this ticket, I though that we may be refactor this code as part of the refactoring ticket which you created other day, but I have done it for this list. Thanks.